CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   <https://github.com/openvinotoolkit/datumaro/pull/550>,
   <https://github.com/openvinotoolkit/datumaro/pull/557>,
   <https://github.com/openvinotoolkit/datumaro/pull/558>)
+- Allowed Pytest-native tests
+  (<https://github.com/openvinotoolkit/datumaro/pull/563>)
```

Comment:
I'm not sure this needs to be in the changelog, since it's not a user-facing change. Up to you, though.

Reply:
I'll remove it from release notes, but it looks as important for developers. Need to think about this again.